.gitignore

```diff
@@ -1,5 +1,6 @@
 settings.json
 *.swp
 .uploads/*
+upload/*
```

Comment:
Why do we need to ignore both `upload` and `.uploads`? Seems redundant.

Reply:
They two different folders that hold profile pictures, which shouldn't be uploaded to Github